### PR TITLE
CE tests and fixes for CE  added

### PR DESCRIPTION
**Pull request by sridevi-15**
#### Fixes #.

- [x] Ran `flake8`
- [x] Ran `run_repo_tests.sh`

#### Changes proposed in this pull request:

- Added tests for sgw-ce and cbs-ce
- Fixed few tests to work with ce

### CODE REVIEW

**Review by mgcouchbase — APPROVED** [review]

File: testsuites/syncgateway/functional/topology_specific_tests/multiple_sync_gateways/test_cache_management_multple_sgws.py [hunk 1]
```diff
@@ -121,7 +122,7 @@ def test_sgw_cache_management_multiple_sgws(params_from_base_test_setup):
             pytest.skip('XATTR tests require --xattrs flag')
         sg_conf_name = "sync_gateway_default_functional_tests"
     else:
-        sg_conf_name = "xattrs/no_import"
+        sg_conf_name = "c"
```
Comment (euniceatcouchbase) [inline testsuites/syncgateway/functional/topology_specific_tests/multiple_sync_gateways/test_cache_management_multple_sgws.py:125]: I am not able to find an sg config file named like c_cc.json under resources/sync_gateway_configs. is this a typo or the sg config missed from the change list?

File: testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_delta_sync.py [hunk 2]
```diff
@@ -712,10 +712,8 @@ def test_delta_sync_larger_than_doc(params_from_base_test_setup, num_of_docs, re
 @pytest.mark.syncgateway
 @pytest.mark.replication
 @pytest.mark.parametrize("num_of_docs, replication_type, file_attachment, continuous", [
-    (10, "pull", None, True),
-    (10, "pull", "sample_text.txt", True),
-    (1, "push", "golden_gate_large.jpg", True),
-    (10, "push", None, True)
+    (10, "pushAndPull", None, True),
+    (10, "pull", "sample_text.txt", True)
```
Comment (euniceatcouchbase) [inline testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_delta_sync.py:716]: I can understand push pull are combined, is push with attachment no longer needed? just curious.

**Review by euniceatcouchbase — CHANGES_REQUESTED** [review]
the PR looks good except one sg_config_name, can you double check if the name correct?

File: testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_delta_sync.py (hunk 2 above)
Comment (sridevi-15) [reply testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_delta_sync.py:716]: this test is specifically for community edition. For regular tests i.e test run with EE/EE already covers with all combinations. For CE delta sync tests, we do not need duplicate testing with all combinations. Just with one attachment and one without attachment is good enough

Comment (euniceatcouchbase) [reply testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_delta_sync.py:716]: sounds good

File: testsuites/syncgateway/functional/topology_specific_tests/multiple_sync_gateways/test_cache_management_multple_sgws.py (hunk 1 above)
Comment (sridevi-15) [reply testsuites/syncgateway/functional/topology_specific_tests/multiple_sync_gateways/test_cache_management_multple_sgws.py:125]: oops,  looks like while fixing flake8 issue, it got replaced with c by mistakenly . Replaced back with right one

**Comment by sridevi-15** [comment]
I found one more test failing with CE. Let me work on the fix and include in the same PR for new tests and then you can check the PR?

**Comment by euniceatcouchbase** [comment]
of course.

**Review by euniceatcouchbase — APPROVED** [review]
looks good to me

**Pull request merged**